lib/model/QubitInformationObject.php

```diff
@@ -2286,10 +2286,13 @@ public function setHistoryByOrigination($history)
 
   public function setLevelOfDescriptionByName($name)
   {
-    // don't proceed with empty $name, or if 'otherlevel' value is passed
+    // Normalize name of "Record group" level of description (for EAD imports)
+    $name = ($name == 'recordgrp') ? sfContext::getInstance()->i18n->__('Record group') : $name;
```

Comment:
The value in the translation messages could be different that the one in the database. Isn't really a way to fix this in the export? Sorry for the insistence.

Reply:
I think the export has to translate human readable terms into an EAD controlled vocabulary (see: https://www.loc.gov/ead/tglib/elements/archdesc.html).

> LEVEL | #REQUIRED, class, collection, file, fonds, item, otherlevel, recordgrp, series, subfonds, subgrp, subseries

And unfortunately there's no constant term ID for the record group level of description term.

I guess what I'll do is look up the term by name with the understanding that if they change the name of the record group level of description in AtoM it's break EAD roundtripping of any record level descriptions.

Reply:
I've made name lookup of the record group LOD localized. Let me know if this seems okay.